### ISSUE TITLE
support copernicus dem in orfeo backscatter

### ISSUE THREAD

**Issue by jdries** [Open-EO/openeo-geopyspark-driver#211]
Copernicus 30m dem is available on creodias, e.g. as COG:
/eodata/auxdata/CopDEM_COG/copernicus-dem-30m/Copernicus_DSM_COG_10_N80_00_W103_00_DEM/Copernicus_DSM_COG_10_N80_00_W103_00_DEM.tif
I believe other formats are there as well.

This will require an upgrade to Orfeo 8.1, which has reworked support for DEM. The support in the current version has proven to be very slow when using certain formats, (e.g. using mapzen tiffs really decreases performance).

Motivation is to also support backscatter above 60°, where srtm is not available.

**Comment by JeroenVerstraelen** [comment]
https://github.com/Open-EO/openeo-geotrellis-kubernetes/commit/9c9c7d77390cbf78dfe2a947148284b3240b5780

**Comment by jdries** [comment]
We'll have to reopen this one: performance was not good, so the solution with the vrt may not work. Other option is to copy over the required dem files locally, like we do for srtm.

**Comment by JeroenVerstraelen** [comment]
The change has been implemented, it now uses a temporary directory containing symlinks to the actual DEM data.

During testing I noticed very slight differences between the backscatter generated with VRTs as DEM compared to when it uses a directory with symlinks. 

VRT output (for La Roche-en-Ardenne)
![Image](https://user-images.githubusercontent.com/3028262/219608007-4a8cda52-3177-4200-a1a4-f18bd0577926.png)

Symlink output
![Image](https://user-images.githubusercontent.com/3028262/219608048-abab041d-43e0-4d11-a8ba-99a016f591c8.png)

The difference is mostly visible when you overlay the images and look at the top of the ridges. The VRT generation also introduces a few more noise-removal pixels.
